src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,31 @@
+package lotto.domain;
+
+import lotto.utils.RandomLottoNumbersGenerator;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class Lottos implements Iterable<Lotto>{
+    List<Lotto> lottoList = new ArrayList<>();
```

Comment:
```suggestion
    private List<Lotto> lottoList = new ArrayList<>();
```
보통 변수 명에는 자료형을 붙이진 않습니다 😄 
왜냐하면 자료형이 변경 되면 변수 명도 변경 되어야 하기 때문입니다 
```java
List<Lotto> lottoList = new ArrayList<>();

// 이럴 경우 변수도 lottoMap으로?? 그렇다면 변수명을 찾아서 다 변경??
Map<Integer, Lotto> lottoList = new HashMap<>(); 
```

그래서 변수명은 형과는 상관 없이 명명하는 편입니다 😅 
이 부분도 참고해주시면 감사하겠습니다 🙇